DRAFT:CDK - Add pp intro doc

EmpieichO opened the pull request:
DRAFT:CDK - Add pp intro + components descriptions doc

Stefan-Ethernal commented on docs/cdk/releases/pp-intro-components.md:
```diff
@@ -0,0 +1,63 @@
+    - `PolygonZkEVMDeployer`
+    - `PolygonZkEVMTimelock`
+- CLI tool: A single command line interface tool for abstracting away the complexity of deploying or configuring CDK components.
+- AggSender is the CDK PP component that accumulates all necessary info in order to generate certificates, which are sent to the SP1 prover via the JSON-RPC API  for the generation of pessimistic proofs.
```
```suggestion
- AggSender is the CDK PP component that accumulates all necessary info in order to generate certificates, which are sent to the SP1 prover via the JSON-RPC API for the generation of pessimistic proofs.
```

mitchpolygon commented:
Hello Anthony! Below you will find changes to the PP docs:

1. Can we make the "AggLayer-side components" a subheading to "Component descriptions"? They shouldn't be on the same level of hierarchy IMO because they are still considered a component description.
1b. I believe there is a spelling error "Here are brief descriptions for each CDK **FEP** component" Should be "Here are brief descriptions for each CDK **PP** component."
<img width="1512" alt="Changes-1-Mitch" src="https://github.com/user-attachments/assets/818ebe93-4757-4262-9f34-47aa7eb2a3af">


2. Update the "See the high level view of the CDK PP chain architecture **here**" link under "AggLayer-side components" to go to the PP chain architecture diagram (currently goes to CDK FEP diagram).
<img width="791" alt="Changes-2-Mitch" src="https://github.com/user-attachments/assets/32491977-bb23-4ff3-a120-28a98b51d90e">

EmpieichO commented:
Hi @mitchpolygon. Thanks for the suggestions.

1. The heading is now adjusted.

2. The link was intentionally and temporarily incorrect because these PRs are separate and their commits are not yet merged. Except for the Review Env of course, so I have corrected it in there.
  ... I will do the same on the actual PR only when finalising the docs for PROD.
